products/green_fast_track/models/_sources.yml

```diff
@@ -121,6 +151,54 @@ sources:
             tests:
               - not_null
 
+      - name: nysshpo_historic_buildings_polygons
+        columns:
+          - name: usnnum
+            tests:
+              - not_null
+          - name: usnname
+            tests:
+              - not_null
+          - name: eligibilitydesc
+            tests:
+            - accepted_values:
```

Comment:
👀 `accepted_values`

Reply:
yup very handy! and DRY so long as the strings aren't needed anywhere else or more useful as structured info like  `nyc_parks_properties_categories.csv`